src/models/ApiMapper.php

```diff
@@ -64,13 +64,10 @@ protected function buildLimit($resultsperpage, $start)
     {
         if ($resultsperpage == 0) {
             // special case, no limits
-            $limit = '';
-        } else {
-            $start = (int)$start;
-            $limit = ' LIMIT ' . $start . ',' . $resultsperpage;
+            return '';
         }
 
-        return $limit;
+        return ' LIMIT ' . intval($start) . ',' . $resultsperpage;
```

Comment:
Why not add the ```intval``` for ```$resultsperpage```?

And why using ```intval``` instead of ```(int)```?

Reply:
Updated to use the int cast as in the original: 9c0ca7d. I had thought that there was extra evaluation within `intval`, but it appears to not be the case. https://3v4l.org/Fmlnk